src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -561,6 +561,43 @@ public void TestDecryptSimple_Pkcs7Signed_GeneratedOnLinux()
             VerifySimpleDecrypt(encodedMessage, certLoader, expectedContentInfo);
         }
 
+        [Fact]
+        [OuterLoop(/* Leaks key on disk if interrupted */)]
+        public void TestDecryptSimpleAes256_RsaOaepSha256()
```

Comment:
Is there an existing OAEP-SHA1 CEK test?

Let's go ahead and make sure we have each of the switch cases covered... so also adding an OAEP-SHA384 and OAEP-SHA512.  If you can convince OpenSSL to produce an OAEP-MD5 I'd be happy to have the exception covered, too.

Reply:
> Is there an existing OAEP-SHA1 CEK test?

Nothing jumped out at me as an existing OAEP-SHA1 test so I added one to be cautious.